src/commands/sensor/commands/deletedatastream.command.ts

```diff
@@ -4,6 +4,6 @@ import { ICommand } from "@nestjs/cqrs";
 export class DeleteDataStreamCommand implements ICommand {
   constructor(
     public readonly id: string,
```

Comment:
Zullen we deze hernoemen naar `aggregateId`? Dat maakt het heel duidelijk wat dit is. `sensorId` is ook een goeie optie (en ik weet niet wat beter is; ik had in Kotlin `sensorId` gebruikt met een `aggregateId()` function in de interface 😉